src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/AADConnectionTest.cs

```diff
@@ -202,9 +202,11 @@ public static void testADPasswordAuthentication()
                     }
                 }
             }
-            catch (SqlException e)
+            catch (SqlException)
```

Comment:
Why bother catching it if all we're going to do is rethrow it?

Reply:
Good point I will remove it.